business/tls.go

```diff
@@ -186,7 +187,7 @@ func (in *TLSService) ClusterWideNSmTLSStatus(ctx context.Context, namespaces []
 		}
 
 		result = append(result, models.MTLSStatus{
-			Status:          mtlsStatus.NamespaceMtlsStatus(namespace.Name).OverallStatus,
+			Status:          mtlsStatus.NamespaceMtlsStatus(namespace.Name, in.businessLayer.App.conf).OverallStatus,
```

Comment:
here as well please

Reply:
arrgh!  I swear I searched for all of these occurrences :-)   Good catch.

Reply:
pushed the fix!  I fixed my regex and searched again and don't see any more occurrences.